packages/ui/src/components/atoms/Toggle.tsx

```diff
@@ -6,47 +6,51 @@ import { goldenGradient } from '../utils/Gradient';
 interface ToggleProps {
   checked: boolean;
   onToggle?: (checked: boolean) => void;
+  discSize?: number;
+  discMargin?: number;
+  width?: number;
+  checkedNode?: React.ReactNode;
+  unCheckedNode?: React.ReactNode;
```

Comment:
All these should be handled internally by the `Toggle` component. The only thing required here is the `variant` and `onText`, `offText`

Reply:
I thought of accepting `ReactNode` instead of `text`, as this component can be easily extended to diplay `svg`/`img`/`text` in future.

Reply:
Created Toggle Variants